### PR TITLE
All changes from the mid-October DNSOP discussion, as separate commits

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: draft-ietf-dnsop-alt-tld.xml [hunk 1]
```diff
@@ -79,7 +79,11 @@
       top-level label can be used as the final (rightmost) label to signify
       that the name is not rooted in the global DNS, and that it should not be
       resolved using the DNS protocol.</t>
-
+      
+      <t>In <xref target="iana-6761"/>, the IANA is requested to add the .alt name
+      to the "Special-Use Domain Name" registry. Such addition will prevent the
+      name from appearing in the global DNS root zone.</t>
```
Comment (wkumari) [inline draft-ietf-dnsop-alt-tld.xml:85]: "Such addition will prevent the name from appearing in the global DNS root zone." - citation needed :-)

I fully agree that from a practical point this is true, but I don't think that there is anything that we can point at that states this clearly. E.g. the last Applicant Guidebook (Sec 2.2.1.2.1 Reserved Names) copied and pasted name from the registry. 
I don't really know to to address this, other than just dropping the sentence or perhaps something more hand-wavey like: "In <xref target="iana-6761"/>, the IANA is requested to add the .alt name to the "Special-Use Domain Name" registry. The expectation is that this name will never be delegated in the global DNS root zone.".... but I suspect that just dropping it is still much better.

File: draft-ietf-dnsop-alt-tld.xml [hunk 2]
```diff
@@ -97,7 +101,7 @@
 
       <section title="Terminology">
         <t>This document assumes familiarity with DNS terms; please see
-        <xref target="RFC8849"/>. Terminology that is specific to this document is:</t>
+        <xref target="RFC8499"/>. Terminology that is specific to this document is:</t>
```
Comment (wkumari) [inline draft-ietf-dnsop-alt-tld.xml:104]: Whargh. Good catch.

File: draft-ietf-dnsop-alt-tld.xml [hunk 3]
```diff
@@ -110,11 +114,10 @@
             <t>non-DNS context: Any other (alternative) namespace.</t>
 
             <t>pseudo-TLD: A label that appears in a fully-qualified domain
-            name in the position of a TLD, but which is not registered in the
+            name in the position of a TLD, but which is not part of the
```
Comment (wkumari) [inline draft-ietf-dnsop-alt-tld.xml:117]: WFM (nice)

File: draft-ietf-dnsop-alt-tld.xml [hunk 4]
```diff
@@ -110,11 +114,10 @@
...
             global DNS. This term is not intended to be pejorative.</t>
 
-            <t>TLD, top-level label: The last visible label in either a fully-qualified domain
-            name or a name that is qualified relative to the root.</t>
+            <t>TLD: See the definition in Section 2 of <xref target="RFC8499"/>.</t>
```
Comment (wkumari) [inline draft-ietf-dnsop-alt-tld.xml:120]: LGTM.

File: draft-ietf-dnsop-alt-tld.xml [hunk 5]
```diff
@@ -160,14 +163,17 @@
...
+      <t>(This paragraph is here to meet the requirements of <xref target="RFC6761"/>.)
+      Application software that uses alternative namespaces in .alt are expected
+      to have their own processing rules for their own names, probably in specialized
+      resolver APIs, libraries, and/or application software.
+      Users might or might not recognize that names in the .alt pseudo-TLD are special.
+      Caching DNS servers and authoritative DNS servers will treat all names in the
+      .alt pseudo-TLD just as they would any other name whose TLD does not appear in the
+      global DNS root.
+      DNS server operators and DNS registries/registrars for the global DNS will never
+      names in the .alt pseudo-TLD because .alt will not be in the global DNS root.</t>
```
Comment (wkumari) [inline draft-ietf-dnsop-alt-tld.xml:176]: Erm. Some comments:
1: "will never names in" - I'm unclear what word is missing, but I'm probably fine with it (perhaps "register")

2:  "will not be in the global DNS root." -> "will not exist in the global DNS root"? 

3: "(This paragraph is here to meet the requirements of <xref target="RFC6761"/>.)" Perhaps "(This paragraph exists to meet the requirements of <xref target="RFC6761"/>, Section 5.)"

Note that the only one I actually care about is 1 (the others are just readability suggestions)

**Review by wkumari — COMMENTED** [review]
Firstly, thank you very much!

I have 2 concerns, see in-line.

File: draft-ietf-dnsop-alt-tld.xml (hunk 5 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-alt-tld.xml]: Fixed by 80a4ddc

File: draft-ietf-dnsop-alt-tld.xml (hunk 1 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-alt-tld.xml]: See revised wording in 1edfc7a

**Comment by vcunat** [comment]
Nit: is it intentional that the link to iana.org is not a (clickable) link?  Visually it's slightly weird to me, at least in the html format: https://www.ietf.org/archive/id/draft-ietf-dnsop-alt-tld-22.html#section-1

**Comment by paulehoffman** [comment]
Good catch. Added PR #45 to fix this.

**Pull request merged**